src/components/layout/Header.vue

```diff
@@ -1,6 +1,7 @@
 <template lang="pug">
   v-toolbar
-    img(src="../../assets/logo.svg", width="150")
+    router-link(:to="{ path: '/' }")
+      img(src="../../assets/logo.svg", width="150")
```

Comment:
Delete the width, add a class to this element and then add the with to that class